### PR TITLE
Chart type cards

### PR DESCRIPTION
**Pull request by designertyler**
Closes https://github.com/carbon-design-system/carbon-website/issues/1263

This adds the grouped bar (horizontal), stacked bar (horizontal), and pie chart tiles to the chart types page.

image

image

### CODE REVIEW

**Comment by theiliad** [comment]
thank you @designertyler this is great!

#987 needs to be merged before this PR since links added in this PR need to point to the new demo link

**Comment by janchild** [comment]
@designertyler Could you please change the page description to the following:

> Start by identifying the purpose of the visualization and then choose the appropriate chart type.

Thank you.

File: src/pages/data-visualization/chart-types/index.mdx [hunk 1]
```diff
@@ -62,6 +62,16 @@ Start by identifying the purpose of the visualization and choose a chart type ap
...
 
+</OverviewCard>
+</Column>
+<Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
+ <OverviewCard
+    title="Grouped bar (horizontal)"
+    href="/data-visualization/basic-charts#grouped-bar"
+    >
+
+![](images/grouped-bar-horizontal.svg)
```
Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:73]: alt here?

Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:73]:
```suggestion
![Grouped bar (horizontal)](images/grouped-bar-horizontal.svg)
```

File: src/pages/data-visualization/chart-types/index.mdx [hunk 2]
```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
     href="/data-visualization/basic-charts#pie"
     >
 
-![Pie](images/donut.svg)
+![](images/pie.svg)
```
Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:143]:
```suggestion
![Pie](images/pie.svg)
```

File: src/pages/data-visualization/chart-types/index.mdx [hunk 3]
```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
...
 </OverviewCard>
 </Column>
+
 <Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
  <OverviewCard
+    title="Stacked bar (horizontal)"
+    href="/data-visualization/basic-charts/#stacked-bar"
+    >
+
+![](images/stacked-bar-horizontal.svg)
```
Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:154]:
```suggestion
![Stacked bar (horizontal)](images/stacked-bar-horizontal.svg)
```

Comment (theiliad) [reply src/pages/data-visualization/chart-types/index.mdx:154]: also is this pointing to the horizontal demo? the link doesn't look like a horizontal demo link

File: src/pages/data-visualization/chart-types/index.mdx [hunk 4]
```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
...
+
+</OverviewCard>
+</Column>
+<Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
+ <OverviewCard
+    title="Stacked area"
+    href="/data-visualization/basic-charts/#stacked-area-chart"
```
Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:161]:
```suggestion
```

File: src/pages/data-visualization/chart-types/index.mdx [hunk 5]
```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
...
+    href="/data-visualization/basic-charts/#stacked-bar"
+    >
+
+![](images/stacked-bar-horizontal.svg)
+
+</OverviewCard>
+</Column>
+<Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
+ <OverviewCard
+    title="Stacked area"
```
Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:160]:
```suggestion
```

File: src/pages/data-visualization/chart-types/index.mdx [hunk 6]
```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
...
+    href="/data-visualization/basic-charts/#stacked-area-chart"
     title="Stacked bar (vertical)"
-    href="/data-visualization/basic-charts/#stacked-bar-(discrete)"
+    href="/data-visualization/basic-charts/#bar-(vertical)"
```
Comment (theiliad) [inline src/pages/data-visualization/chart-types/index.mdx:163]: this doesn't look like it'll actually scroll down to the stacked bar demo, it'll go to the **Bar (vertical)** heading

**Comment by jeanservaas** [comment]
Looks good, just take design tag off of these two, they're live now!

image

**Comment by mjabbink** [comment]
@designertyler We need to get this wrapped.

For my understanding
— Make image tile for Meter (with design only tag)
— Remove <Design only> tag from Gauge and stacked area chart
— Remove Stacked area chart content from Design only section (Only Meter to remain)

—Create a image card for Meter in another issue. Meter soon to be released.


Did I miss anything? Maybe Histogram? I believe that is in our near future.

**Comment by designertyler** [comment]
@mjabbink This PR was to just add the grouped bar (horizontal), stacked bar (horizontal), and pie chart tiles on the chart overview page. I think the other additions were being worked on when this was opened.

We do have an issue to make an image tile for meter here https://github.com/carbon-design-system/carbon-website/issues/1355 that would separate the meter/gauge tiles and remove the tags.

I'll get a PR opened to remove the duplicate content under "Design only" and the stacked area shouldn't have a tag on the Chart types page.

**Comment by mjabbink** [comment]
👍

**Pull request merged**